reference/7.1/Microsoft.PowerShell.Core/About/about_Splatting.md

```diff
@@ -151,6 +154,8 @@ Hello World!
 
 ## Examples
 
+### Example 1
+
```

Comment:
Need to add descriptive titles to the example H3 lines

Reply:
Fixed